docs/faq.rst

```diff
@@ -187,6 +187,20 @@ all the part files and manually concatenate them together.
 
 For more details, see :ref:`checkpoints_parts`.
 
+My repository is corrupted, how can I restore from an older but working copy?
+-----------------------------------------------------------------------------
+
+The recommended method is to delete the corrupted repository, and then copy the
+working repo to the same location:
+
+::
+
+    borg delete /path/to/repo
+    cp -avr /path/to/repo-good /path/to/repo
```

Comment:
There is a nonce-reuse security issue with that.

Reply:
How do you mean that? Isn't the local .config/borg/security/.../nonce deleted by ``borg delete``?

Reply:
Guess so. And that is (IIRC) the problem. If there is no local information about the highest already used counter (nonce), it will rely on the counter determined from the repo.
And if you use an older copy of the repo, the counter will be lower and thus it will reuse counter values.

Reply:
So this should be considered an issue in ``borg delete`` and as a work-around until it is fixed, we could add a ``rm -r ~/.config/borg/security/<ID>`` (or just <ID>/nonce?)? If you agree, I'd open an issue for that and add this to the PR.

Reply:
not sure you understood my previous comment correctly.

it would need rather the opposite of `rm` to work around, I guess.

Reply:
What is bad about reusing older nonces?

Reply:
it breaks the security of AES-CTR encryption.

Reply:
OK, so we have to use the higher nonce of the deleted repo with the older repo. How to achieve that? Maybe rename the security/<ID> directory so that ``borg delete`` doesn't remove it? And rename it back finally?

Reply:
yeah, could be. but first try that practically.

Reply:
OK, I will improve it and run a test.